### PR TITLE
CANS-664 Update styles for tooltip hover text

### PR DESCRIPTION
**Pull request by nasar-cambria**
https://osi-cwds.atlassian.net/browse/CANS-715

## Description
Update tooltip hover text styles to match the recommended design approach.

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Assessment/style.sass [hunk 1]
```diff
@@ -6,10 +6,12 @@ $item-confidential-label-height: 6.05rem
 //********************
 .assessment-tooltip
   background-color: #1a1a1a
+  font-weight: normal
   color: #fff
-  font-size: 1.4rem
+  font-size: 15px
   text-align: left
   width: 30rem
+  line-height: 20px
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/Assessment/style.sass]: We should use `1.5rem` and `2rem` here, instead of `px`.

Comment (wmitchell-cambria) [reply app/javascript/Application/components/Assessment/style.sass]: To clarify, we should almost never be using `px`. Exceptions include:
* Small things like borders or box-shadows which don't affect overall layout.
* Icons or images which are already a fixed pixel size and need small adjustments for visual balance.

Comment (nasar-cambria) [reply app/javascript/Application/components/Assessment/style.sass]: Sure, I'll change it.

**Comment by nasar-cambria** [comment]
Jenkins re-test this please

**Pull request merged**